Makefile

```diff
@@ -75,8 +75,7 @@ clean: clean-protoc cleanall-cli clean-server clean-beat clean-agent clean-monit
 DOCKER_CMD ?= "docker"
 
 build-base: protoc build-server build-gateway build-beat build-agent build-monit
-build-local-plugin: build-amp-local build-ampagent
```

Comment:
this target was used by the Travis build, no need to build the aws plugin (and soon the azure plugin, etc) when we can only test locally.

Reply:
even if we're not actually using them, it's still good to make sure they actually build

Reply:
sure, but the build time is increasing significantly